### PR TITLE
Add blog post about OCI security profiles

### PR DESCRIPTION
**Pull request by saschagrunert**
PTAL @jhrozek @JAORMX @pjbgf @ccojocar 

Targeted for May 24.

### CODE REVIEW

**Comment by sftim** [comment]
> Targeted for April 12, right before KubeCon.

Sorry, we have the Kubernetes post-release communication series of blog announcements coming up. You'll need to liaise with SIG Release's comms team to get _that_ date. The Kubernetes v1.27 release is likely to happen on the 11th of April.

/hold

pending allocation of publication date.

**Comment by saschagrunert** [comment]
@kubernetes/sig-docs-blog-owners would a release date this week work for us?

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (jhrozek) [inline content/en/blog/_posts/2023-04-05-oci-security-profiles.md:84]: Is it up to 24 hours and up to 1000 profiles? What happens when there's more than 1000 profiles?

**Review by jhrozek — COMMENTED** [review]
one small nit inline..

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-04-05-oci-security-profiles.md:84]: Yep, both limitations will apply. The oldest will be dropped and we will re-cache it.

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (tuxerrante) [inline content/en/blog/_posts/2023-04-05-oci-security-profiles.md:191]: On all cluster nodes?

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-04-05-oci-security-profiles.md:166]: You can use the ORAS CLI for pulling the artifact: https://oras.land/cli

Docker does not support OCI artifacts right now. :)

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-04-05-oci-security-profiles.md:191]: No just on the local machine.

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (pjbgf) [inline content/en/blog/_posts/2023-04-05-oci-security-profiles.md:50]:
```suggestion
versioned, namespaced and annotated similar to regular container images.
```

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (pjbgf) [inline content/en/blog/_posts/2023-04-05-oci-security-profiles.md:110]:
```suggestion
are a bunch of registries that already supports OCI artifacts:
```

**Review by pjbgf — COMMENTED** [review]
A few tiny nits, otherwise it looks good. Great job @saschagrunert! :clap:

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (sftim) [inline content/en/blog/_posts/2023-04-05-oci-security-profiles.md:54]:
```suggestion
To see how that works in action,  specify a `baseProfileName` prefixed
```
We [avoid using we](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we) in tech docs and in other contexts where it's not clear who the “we” refers to.

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (sftim) [inline content/en/blog/_posts/2023-04-05-oci-security-profiles.md:89]: Try this:
```suggestion
Because the overall resulting syscalls are hidden from the user (I only have the
`baseProfileName` listed in the SeccompProfile, and not the syscalls themselves), I'll additionally
annotate that SeccompProfile with the final `syscalls`.

Here's how the SeccompProfile looks after I annotate it:
```

---

**Aside**
This is a private annotation - is this just a hint to the humans operating the workload?
If the annotation is intended to be machine readable / consumed elsewhere that's an architecture bug.
Kubernetes code should use public annotations (eg `spo.x-k8s.io/allowed-system-calls`).

**Review by sftim — COMMENTED** [review]
Thanks. Here's some feedback on style, and a query about a private (?) annotation.

File: content/en/blog/_posts/2023-04-05-oci-security-profiles.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-04-05-oci-security-profiles.md:89]: > This is a private annotation - is this just a hint to the humans operating the workload?

Yes, nothing machine readable.

**Comment by saschagrunert** [comment]
Thanks @sftim, fixed everything as suggested.

**Comment by sftim** [comment]
I'm afraid this'll need a new publication date. How does late May sound? If you want publication earlier, you can; please liaise with release comms.

**Comment by saschagrunert** [comment]
@sftim may sounds good to me! Do you have a specific date in mind?

**Comment by saschagrunert** [comment]
@sftim friendly ping, which date do we want to choose?

**Comment by saschagrunert** [comment]
@sftim I moved that one to 2023-05-24 for now if that would be a good fit.

**Comment by nate-double-u** [comment]
Hi @saschagrunert, May 24th looks good to me, it doesn't look like it's conflicting with anything else on [our blog calendar](https://github.com/orgs/kubernetes/projects/93/views/3).

**Comment by nate-double-u** [comment]
This still needs an lgtm -- @pjbgf, @jhrozek, @tuxerrante, since you reviewed earlier, could you take another look to make sure your comments were addressed adequately and provide an lgtm if so?

**Comment by tuxerrante** [comment]
Totally fine for me 🙂
/lgtm

**Review by jhrozek — APPROVED** [review]
/lgtm

**Comment by saschagrunert** [comment]
/unhold

@nate-double-u may I ask you to approve?

**Comment by sftim** [comment]
Thanks

/approve

**Pull request merged**
